src/iam/chainConfig.ts

```diff
@@ -1,18 +1,24 @@
 import { VoltaAddress1056 } from "@ew-did-registry/did-ethr-resolver";
 import { CacheServerClientOptions } from "../cacheServerClient/cacheServerClient";
+import { MessagingMethod } from "../utils/constants";
```

Comment:
tbh, I'm a confused by the purpose of the `MessagingMethod` enum. It seems like we are listing `CacheServer`, `WebRTC` and `SmartContractStorage`, but we have only implemented `CacheServer`? If so, what's the point of having the other ones?

Reply:
Other ones as @manihagh said are going to be implemented in the future.

Reply:
Ok, thanks.